### DISCUSSION TITLE
My sketchybar has ? icons instead of actual images

### DISCUSSION THREAD

**Discussion by shndowbots** [FelixKratz/SketchyBar#263] [Q&A] [answered]
My sketchybar is only showing ? icons instead of actual images, I don't think this is a config problem, since it has this icon for multiple different configs. I'm using this config: https://github.com/shahmilav/carbonfiber-dots/tree/main/sketchybar
<img width="1417" alt="Screen Shot 2022-10-18 at 6 25 10 PM" src="https://user-images.githubusercontent.com/93618275/196575475-236200cb-905d-44a0-82b3-0dd99bad85c9.png">

Any ways to fix this?
On Mac M1 btw.
Thank you!

**Comment by FelixKratz** [comment, marked as answer]
You need to install the fonts for those configs. 

A common source of icons is apples sf-symbols font:

```bash
brew install --cask sf-symbols
```

And the default font of sketchybar is the Hack Nerd Font:
```bash
brew tap homebrew/cask-fonts
brew install --cask font-hack-nerd-font
```

Be sure to install those and any additional fonts that the config may use.

**Reply by shndowbots** [reply]
Thank you!
